business-central-parent/add-ons-distribution/target/antrun/build-main.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project name="maven-antrun-" default="main"  >
+<target name="main">
+  <copy includeemptydirs="false" todir="/home/emingora/development/projects/RedHat/issues/BXMSPROD-598/kie-wb-distributions/business-central-parent/add-ons-distribution/target/classes" failonerror="false" quiet="true">
```

Comment:
Was this file left by mistake?

If not, I suspect the paths are wrong.

Reply:
Sure, .gitignore file added to add-ons-distribution module and target folder removed. Thanks @manstis